tests/integration/is-empty-test.js

```diff
@@ -0,0 +1,162 @@
+import { isEmpty } from 'ember-awesome-macros';
+import { module, test } from 'qunit';
+import { A as emberA } from '@ember/array';
+import compute from 'ember-macro-test-helpers/compute';
+
+module('Integration | Macro | isEmpty');
+
+test('empty without params', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty(),
+    properties: {},
+    strictEqual: true
+  });
+});
+
+test('empty with null', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: null
+    },
+    strictEqual: true
+  });
+});
+
+test('empty with undefined', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: undefined
+    },
+    strictEqual: true
+  });
+});
+
+test('empty with empty string', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: ''
+    },
+    strictEqual: true
+  });
+});
+
+test('empty with empty array', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: []
+    },
+    strictEqual: true
+  });
+});
+
+test('empty with object size 0', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: { size: 0 }
+    },
+    strictEqual: true
+  });
+});
+
+test('not empty with empty object', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: {}
+    },
+    strictEqual: false
+  });
+});
+
+test('not empty with string', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: 'Adam Hawkins'
+    },
+    strictEqual: false
+  });
+});
+
+test('not empty with array', function(assert) {
+  compute({
+    assert,
+    computed: isEmpty('source'),
+    properties: {
+      source: [0, 1, 2]
+    },
+    strictEqual: false
+  });
+});
+
+test('not empty with eol & tab', function(assert) {
```

Comment:
Most of these tests can be removed. We don't need to test the implementation of Ember.isEmpty, just that our macro works in ways we expect.